pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -516,6 +577,20 @@ func (vm *windows) getSourceVIP() (string, error) {
 	return sourceVIP, nil
 }
 
+// newRemoteFileInfo returns a pointer to a FileInfo object created from the specified file on the Windows VM
+func (vm *windows) newRemoteFileInfo(path string) (*files.FileInfo, error) {
```

Comment:
`vm` implies remote. I suggest naming this `newFileInfo()`. Though I am wondering if there is a Go type embedding friendly way of doing this that is not immediately obvious to me. Maybe @LorbusChris knows.